src/nginx/grpc.cc

```diff
@@ -76,6 +76,42 @@ std::pair<Status, std::string> GrpcGetBackendAddress(
       Status(NGX_DECLINED, "No GRPC backend address specified"), std::string());
 }
 
+std::shared_ptr<::grpc::ChannelCredentials> CreateChannelCredentials(
+    ngx_http_request_t *r, ngx_esp_loc_conf_t *espcf) {
+  if (espcf->grpc_backend_ssl == nullptr) {
+    ngx_log_error(NGX_LOG_DEBUG, r->connection->log, 0,
+                  "GrpcGetStub: Use insecure channel credentials");
+    return ::grpc::InsecureChannelCredentials();
+  }
+
+  if (espcf->grpc_backend_ssl->use_google_default) {
+    ngx_log_error(NGX_LOG_DEBUG, r->connection->log, 0,
```

Comment:
Should this be a Error log?

Reply:
Not this is a valid use case, not an error.